internal/cli/server/chains/amoy.go

```diff
@@ -59,6 +59,10 @@ var amoyTestnet = &Chain{
 					"0":     "0x000000000000000000000000000000000000dead",
 					"73100": "0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9",
 				},
+				Coinbase: map[string]string{
+					"0":        "0x0000000000000000000000000000000000000000",
+					"26201856": "0x000000000000000000000000000000000000ba5e",
```

Comment:
`0x000000000000000000000000000000000000ba5e` is just a place hold. The final address is TBD.